spec/views/post_index_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe 'Post view', type: :feature do
```

Comment:
* I suggest you add these missing tests as they are required 
* ` Tests that asserts you can see the user's profile picture.`